ktlint.gradle

```diff
@@ -0,0 +1,25 @@
+repositories {
```

Comment:
we probably don't need to re-add repositories here. The parent gradle file will provide jcenter.

BUT

if we go with the ktlint gradle plugin, this entire file could be omitted.

Reply:
Fixed, removed the repository declaration here.